init.sh

```diff
@@ -78,8 +78,18 @@ if [[ ! -f lup-${NAME} ]]; then
     sudo cp lup-${NAME} /etc/sudoers.d/
 fi
 
-[[ -d lupmgr ]] || git clone https://github.com/lupfoss/lupmgr.git
-cd lupmgr && git pull && git checkout ${BRANCH}
+if [[ -z "${LIGHTUP_TAR_GZ_VERSION}" ]]; then
+    echo "cloning lightup manager"
+    [[ -d lupmgr ]] || git clone https://github.com/lupfoss/lupmgr.git
+    cd lupmgr && git pull && git checkout ${BRANCH}
+else
+    echo "getting lightup manager"
+    LIGHTUP_TAR_GZ_TARGET="v${LIGHTUP_TAR_GZ_VERSION}.tar.gz"
+    wget https://github.com/lupfoss/lupmgr/archive/refs/tags/${LIGHTUP_TAR_GZ_TARGET}
```

Comment:
fyi we use curl for all of our other downloads. wget is probably going to be installed too but might not be guaranteed

Reply:
Yes. Let's use curl 
It makes sense.